### PR TITLE
 main view fix

### PR DESCRIPTION
**Pull request by bergano65**
link learn more by my opinion is informative in main view so decided not remove it

### CODE REVIEW

File: CogsMinimizer/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -34,7 +34,7 @@
     {
         <div class="col-lg-4 thumbnail">
             <div class="caption">
-                <h3>Sign-in to connect your Azure Subscription</h3>
+                <h3>Sign-in to connect to your Azure Subscription</h3>
```
Comment (shklar) [inline CogsMinimizer/Views/Home/Index.cshtml]: There is a conflict in this file.
Please resolve it.

**Comment by bergano65** [comment]
Ok, I’ll synchronize sources and will send again. Afterwards will fix message for reset resources, wait day before subscription will be analyzed and my resources filtering issue. In this order. Afterwards will remain only analyze view fix (most extensive fix) and collecting statistics for subscriptions feature.

**Comment by bergano65** [comment]
To simplify merge and avoid excessive merge in history of main branch I’m deleting pull request and branch and will submit fix over existing main branch state.

**Comment by bergano65** [comment]
Pls look submitted pull request. Minor changes just to finish main view fixes.

From: Evgeny Vitenberg
Sent: Sunday, December 25, 2016 1:53 AM
To: shklar/SubMinimizer <reply@reply.github.com>; shklar/SubMinimizer <SubMinimizer@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [shklar/SubMinimizer] main view fix (#49)

To simplify merge and avoid excessive merge in history of main branch I’m deleting pull request and branch and will submit fix over existing main branch state.

**Pull request closed without merging**